packages/sentry/src/sentry.ts

```diff
@@ -45,9 +45,12 @@ export const boot = () => {
   }
 };
 
+const isEntras = (extrasOrExtra: Extra | Extras): extrasOrExtra is Extras =>
+  typeof extrasOrExtra === 'object';
+
 export const reportErrorToSentry = (
   error: Reportable,
-  extraInfo?: { extra?: object | string },
+  extraInfo?: { extra?: Extra | Extras },
```

Comment:
Sentries internal types have changed. I think using them directly is sensible anyway. Additionally using a Type Guard to to the check makes the usage type safer.